app/models/tv_episode.rb

```diff
@@ -0,0 +1,35 @@
+class TVEpisode
```

Comment:
This file has the same pattern or refactoring:
* rename to singular version
* remove instance variables
* create `parse_record` and `parse_records` methods